docs/fixes/2026-02-17-aws-default-profile-sso-interference.md

```
@@ -0,0 +1,121 @@
+credential processes, or other non-trivial settings, the SDK attempts to resolve those during
+`LoadDefaultConfig` and may fail with a confusing error.
+
+## Symptoms
+
+```
+Error: failed to load AWS config
+
+## Explanation
+Failed to load AWS configuration for SSO authentication in region 'us-west-2'
+
+## Hints
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)